README.md

```diff
@@ -189,6 +189,38 @@ We recommend querying using payment key hashes (`addr_vkh`) when possible (other
       |}>
     |}
   }
+ ```
+</details>
+<details>
+  <summary>api/pool/delegationHistory</summary>
+  Input
+
+  ```js
+  {
+    poolRanges: Dictionary<string, {fromEpoch: number, toEpoch?: number}> // operator key (pool id)
```

Comment:
Should probably mention whether or not these are inclusive

Reply:
yeah good suggestion (for the record both of them are)